api/scpca_portal/models/library.py

```diff
@@ -198,9 +198,16 @@ def get_combined_library_metadata(self) -> List[Dict]:
         for sample in self.samples.all():
             metadata = self.project.get_metadata() | sample.get_metadata() | self.get_metadata()
 
-            # Only single cell multiplexed sample libraries should pass through sample_cell_estimate
-            if not self.is_multiplexed:
-                del metadata["sample_cell_estimate"]
+            # NOTE: demux_cell_count_estimate is reassigned here for the output metadata file.
+            # The metadata file writes out sample-library rows, necessitating greater granularity.
+            # In contrast, the Sample model expects the aggregate value of its related libraries.
+            if self.is_multiplexed:
+                metadata["demux_cell_count_estimate"] = self.metadata["sample_cell_estimates"][
+                    sample.scpca_id
+                ]
+            # Only multiplexed sample-libraries should have a demux_cell_count_estimate attribute
+            else:
+                del metadata["demux_cell_count_estimate"]
```

Comment:
An alternative to deleting this here would be to just delete `demux_cell_count_estimate` from `derived_properties` in `Sample::get_metadata`. We're not treating it as a derived property here as we're grabbing the value from elsewhere, so it could be confusing to add it as a derived property only to repopulate it or delete it right thereafter.

Reply:
Yeah, let's remove it from the `Sample.get_metadata` `derived_attributes` since it should not exist in the metadata. Additionally, we should just rename `Sample.demux_cell_count_estimate` to be `demux_cell_count_estimate_sum`. Note we will need to update the client code as well so that the samples table renders correctly.

Reply:
Files to be updated:
- Migration, serializer, expected values, client side references